client/src/components/Input/SubmitButton.jsx

```diff
@@ -38,7 +38,7 @@ export default function SubmitButton({
         </div>
       </button>
     );
-  } else if (!isTokenProvided && (endpoint !== 'openAI' || endpoint !== 'azureOpenAI' )) {
+  } else if (!isTokenProvided && endpoint !== 'bingAI' && (endpoint !== 'openAI' || endpoint !== 'azureOpenAI')) {
```

Comment:
logic for this is kind of messy so I will be refactoring in an upcoming PR

Reply:
I agree, thanks!